src/generated/resources/data/occultism/recipe/ritual/possess_villager.json

```diff
@@ -0,0 +1,52 @@
+{
+  "type": "occultism:ritual",
+  "activation_item": {
+    "item": "occultism:book_of_binding_bound_djinni"
+  },
+  "entity_tag_to_summon": "c:villagers",
+  "ingredients": [
+    {
+      "tag": "c:buckets/water"
+    },
+    {
+      "tag": "c:buckets/milk"
+    },
+    {
+      "tag": "c:storage_blocks/coal"
+    },
+    {
+      "tag": "c:storage_blocks/coal"
+    },
+    {
+      "tag": "c:storage_blocks/bone_meal"
+    },
+    {
+      "tag": "c:storage_blocks/bone_meal"
+    },
+    {
+      "item": "minecraft:soul_sand"
```

Comment:
Some of these don't feel very fitting to a villager summon, like soul sand and quartz. I'd suggest some simpler ingredients like some tools and seeds. A hoe, axe, and 4 seeds would be very suitable.

Reply:
In occultism villagers there is a tag "human" this recipe is a reference to the chemical compounds of the human body (soulsand is because villagers probably have souls), maybe i'll refine this

Reply:
changed to a bed, campfire and pie to kindly invite the villager into your home